### ISSUE TITLE
Switch pro Controller on xbox game pass

### ISSUE THREAD

**Issue by durza500** [Davidobot/BetterJoy#668]
Hey guys,  i installed betterjoy and it worked perfectly for me but only on Epic games. So i tried to use  it on Xbox game pass to be more specific in ori and the blind forest and Doom eternal but it didnt  recognize  my Controller any ideas? Thank You

**Comment by CamberLoid** [comment]
Can confirm on me.

version 7.0

**Comment by Link-HXX** [comment]
Yes,I found that it doesn’t work in most games, it only works in Forza Horizon 5.  version 7.1

**Comment by rubberto** [comment]
Same here, tested on Hades and it worked properly for a few seconds before going back to the way it was before. Version 7.1

**Comment by skye1384-wq** [comment]
Same problems here for Ori and the blind forest
